sklearn/cluster/optics_.py

```diff
@@ -753,16 +748,16 @@ def _xi_cluster(reachability_plot, predecessor_plot, ordering, xi, min_samples,
         reachability plot is defined by the ratio from one point to its
         successor being at most 1-xi.
 
-    min_samples : int > 1 or float between 0 and 1 (default=None)
-        The same as the min_samples given to OPTICS. Up and down steep regions
-        can't have more then ``min_samples`` consecutive non-steep points.
-        Expressed as an absolute number or a fraction of the number of samples
-        (rounded to be at least 2).
+    min_samples : int > 1 (default=None)
+        The same as the min_samples given to OPTICS when it was an int. In case
+        a float was given to OPTICS it's the same but scaled. Up and down steep
```

Comment:
"In case a float was given" is contradicting the dtype that says that it is an int.